openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_scriptsmenu.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "dict",
+  "collapsible": true,
+  "key": "scriptsmenu",
+  "label": "Scripts Menu Definition",
+  "children": [
```

Comment:
Could we change this to modifiable dicationary (key == label, value definition of menu under label). This way studio could add their custom menu as they would like to.

Reply:
This is suggestion and not required in this PR.